### PR TITLE
OBG 1034 tiny controller for consent retrieval

### PR DESCRIPTION
**Pull request by electronicpeter**

### CODE REVIEW

File: fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml [hunk 1]
```diff
@@ -416,6 +416,28 @@ paths:
         "401":
           $ref: "#/components/responses/401_Unauthorized"
 
+  /v1/consent/{userid}/{password}:
```
Comment (valb3r) [inline fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml:419]: Can we make password to be in request header? While the controller is for debug, still we can send password properly

File: fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml [hunk 2]
```diff
@@ -416,6 +416,28 @@ paths:
...
+    get:
+      summary: ask for existing consent of user
```
Comment (valb3r) [inline fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml:421]: Add note that it is a temporary endpoint for debug

**Review by valb3r — APPROVED** [review]

File: fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml (hunk 2 above)
Comment (electronicpeter) [reply fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml:421]: Its written in description, that is depends on spring property.

File: fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml (hunk 1 above)
Comment (electronicpeter) [reply fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml:419]: it should be called by browser, not http client. we can remove password completly.

**Pull request merged**